webapp/components/ContributionForm/ContributionForm.vue

```diff
@@ -204,6 +216,9 @@ export default {
     ...mapGetters({
       currentUser: 'auth/user',
     }),
+    showDeleteImageButton() {
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-02-18T16:26:30Z" title="Tuesday, February 18th 2020, 5:26:30 pm +01:00">Feb 18, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -204,6 +217,12 @@ export default {
     ...mapGetters({
       currentUser: 'auth/user',
     }),
+    showDeleteImageButton() {
```


Testing the existence of `showDeleteImageButton` is a great test case.